### DISCUSSION TITLE
Thoughts on adding support for a custom.conf in addition the global.conf?

### DISCUSSION THREAD

**Discussion by albino1** [KBlixt/subcleaner#42] [Ideas]
Right now you can add your own non-language specific keywords and regex in the `global.conf` file, but when you update to the latest version of subcleaner that file gets replaced.  That makes sense, and I want your updated changes, however that doesn't leave many options for adding our own custom filters other than re-adding them to the `global.conf` file each time.

So I wonder if it might be worth allowing us to add some sort of `custom.conf` that applies to all subtitle files where we can place our own things to be cleaned and that won't get overwritten when updating.

Thanks!

**Comment by KBlixt** [comment]
Any regex profile outside the default folder is your own regex profiles that won't be overwritten, there you can add the custom.conf and specify which languages it should run against.

Read the Readme file in the regex directory, hopefully it should clarify.

But from what I gather from your post, create a global2.conf and add your own regex there and then it will run along side any default regex that might get updated later.

**Reply by albino1** [reply]
Ah, I see.  I assumed it would only run against a single conf for a specified language, but I see now actually reading it properly that I can just make one called anything I want with no language code specified and it will apply to everything.  Thanks!

I don't know of a way to close a discussion, but feel free to close it if you can.
